src/components/Checkout.js

```diff
@@ -1,5 +1,52 @@
-const Checkout = () => {
-  return <p>Checkout here</p>
+import Form from './Form'
+import { useNavigate } from 'react-router-dom'
+import '../styles/_Checkout.scss'
+
+const Checkout = ({shoppingBag, totalPrice}) => {
+
+  const navigate = useNavigate()
+
+  if(!shoppingBag.length) {
+    navigate('/shopping-bag')
+  }
+
+  const items = shoppingBag.map(item => {
+    return <div className='request-item' key={item.id}>
+      <div>
+        <b>{item.quantity}x {item.type}</b>
+        <p>Color: {item.color}</p>
+        {item.size !== 'onesize' && <p>Size: {item.size}</p>}
+      </div>
+      <b>${(item.quantity * item.price)}</b>
+    </div>
+  })
+
+  const numberOfItems =
+    shoppingBag.reduce((acc, curr) => {
+      return acc + curr.quantity
+    }, 0)
+
+  return (<div className="checkout">
+    <div className="checkout-header">
+      <h2>EverUse</h2>
+      <h3>Order Request</h3>
+    </div>
+    <p>
+    Requests will be sent to EverUse and followed up within 5 business days. Payment through (methods) will be discussed over email.
```

Comment:
Would love to know your and team's thoughts on `Payment options will be discussed over email`.

Reply:
i'm down to change it to something more specific, didn't put too much thought into it used what was on the wireframe

Reply:
Same. Didn't think about it much when I wrote it up. That sounds good Taranveer.